build.gradle

```diff
@@ -32,19 +32,27 @@ if (System.getenv('CI') == 'true') {
   gradle.startParameter.showStacktrace = ShowStacktrace.ALWAYS
 }
 
-import org.ajoberstar.grgit.Grgit
-
 allprojects {
-  apply plugin: 'org.ajoberstar.grgit'
-  def git = Grgit.open(currentDir: rootDir)
-  def maybeSnapshot = System.getenv('CIRCLE_BRANCH') == 'master' ? '' : '-SNAPSHOT'
-  def shortCommitHash = "+git.${git.head().abbreviatedId}"
-  def maybeDirty = git.status().clean ? '' : '-dirty'
-  git.close()
-
   group 'com.virtuslab'
+
   apply from: "$rootDir/version.gradle"
-  version "${PLUGIN_VERSION}${maybeSnapshot}${shortCommitHash}${maybeDirty}"
+  def ciBranch = System.getenv('CIRCLE_BRANCH')
+  if (ciBranch == 'master') {
+    version PLUGIN_VERSION
```

Comment:
No longer attach build metadata to released versions, we'll have tags to identify them instead